.github/workflows/gh-registry.yml

```diff
@@ -70,3 +70,20 @@ jobs:
         working-directory: 'hello-strongbox-npm'
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  publish-hello-strongbox-maven:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-java@v1
+        with:
+            java-version: 1.11
+      - name: 'Copy Settings'
+        run: cp ~/.m2/settings.xml settings.xml
```

Comment:
Question: Where is this `settings.xml` coming from and what are its contents? Could you please provide links to it and the documentation?

Reply:
Yeah I think this shouldn't be necessary. The `pom.xml` should be updated to point to the GitHub Package Registry similar to gradle (build.gradle).

https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-apache-maven-for-use-with-github-packages#publishing-a-package

Reply:
Although GH says, the `settings.xml` and `pom.xml` don't have to be updated at all IMHO.

You should be able to simply do:
```
mvn deploy -DaltDeploymentRepository=github::default::https://maven.pkg.github.com/OWNER/REPOSITORY
``` 

And that should work.

Reply:
I was getting worried due to the repositories not available (e.g. localhost), but it works by applying the altDeploymentRepository. However i have some problems reproducing the deployment locally or in my fork. Shouldn't be that big of a deal though. I agree the sed solution was only temporary :).